tools/generator/utils/ioext.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package utils
+
+import (
+	"io"
+	"io/ioutil"
+	"strings"
+
+	"github.com/ahmetb/go-linq/v3"
+)
+
+func GetLines(file io.Reader) ([]string, error) {
```

Comment:
We actually have the `ioext` package in `tools/internal/ioext`, please consider move this function there

Reply:
fixed